readme.md

```diff
@@ -120,17 +120,19 @@ You can pass a query parameter `&hide=` to hide any specific stats with comma-se
 
 ### Adding private contributions count to total commits count
 
-You can add the count of all your private contributions to the total commits count by using the query parameter `&count_private=true`.
-
-> **Note**
-> If you are deploying this project yourself, the private contributions will be counted by default. If you are using the public Vercel instance, you need to choose to [share your private contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile).
+You can add the count of all your private contributions to the total commits count by using the query parameter `&count_private=true`. To enable this, you need to choose to [share your private contributions](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/managing-contribution-settings-on-your-profile/showing-your-private-contributions-and-achievements-on-your-profile).
 
```

Comment:
Is this necessary? The initial docs mentions "contributions" instead of "commits" too, and there doesn't seem to be much of a difference between what was initially there and what you wrote. Your choice though!

Reply:
The changes in this file are a restructure of the existing doc, plus some new information added based on what @rickstaa shared in https://github.com/anuraghazra/github-readme-stats/issues/2500.

Sorry if this is too much being changed, or makes it look like I'm belittling what others have previously written. I think the current documentation is great, the restructure in this PR was an attempt to avoid duplicating any info so that it still reads/flows well.

Happy to change anything however you'd prefer.

Reply:
No issues; I just wondered why you decided to make that change! I'm okay with either version, so it's entirely up to you. We'll see what @rickstaa thinks...